.agents/skills/afk-team-workflow/references/spawn-prompts.md

```
@@ -0,0 +1,35 @@
+# Spawn prompt 模板
+
+按阶段取用，填入变量。契约文件路径必须填**主仓库的绝对路径**：teammate 的 worktree 基于 origin/main 检出，未必包含（最新的）skill 文件。
+
+## 实现
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)